actions/admin/settings/150.mail.php

```diff
@@ -139,6 +139,26 @@
 					'save_method' => 'storeSettingField',
 					'advanced_mode' => true
 				],
+                'system_mda_conf_dir' => [
+                    'label' => lng('serversettings.mda_conf_dir'),
+                    'settinggroup' => 'system',
+                    'varname' => 'mda_conf_dir',
+                    'type' => 'text',
+                    'string_type' => 'filedir',
```

Comment:
Use string_type `confdir` here as this setting will not be a file and we want to check for disallowed paths, also, the (default) value should end with `/`